### ISSUE TITLE
Failure toinstall[Bug]: 

### ISSUE THREAD

**Issue by dafejimi** [netlify/next-runtime#2397]
### Summary

i keep running into this error:npm ERR! code 1
npm ERR! path C:\Users\HP\Desktop\Work Folder\upwork-assessment\node_modules\ipx\node_modules\sharp
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c (node install/libvips && node install/dll-copy && prebuild-install) || (node install/can-compile && node-gyp rebuild && node install/dll-copy)
npm ERR! sharp: Downloading https://github.com/lovell/sharp-libvips/releases/download/v8.14.5/libvips-8.14.5-win32-x64.tar.br
npm ERR! sharp: Please see [sharp - High performance Node.js image processing](https://sharp.pixelplumbing.com/install) for required dependencies

npm ERR! sharp: Installation error: Request timed out

### A link to a reproduction repository

N/A

### Expected Result

Complet installation

### Actual Result

Errors

### Steps to reproduce

N/A

### Next Runtime version

4.41.3

### Is your issue related to the `app` directory?

- [ ] Yes, I am using the `app` directory

### More information about your build

- [ ] I am building using the CLI
- [ ] I am building using file-based configuration (`netlify.toml`)

### What OS are you using?

Windows

### Your netlify.toml file

<details>
  <summary>`netlify.toml`</summary>

```toml
# Paste content of your `netlify.toml` file here
```

</details>

### Your public/_redirects file

<details>
  <summary>`_redirects`</summary>

```
# Paste content of your `_redirects` file here
```

</details>

### Your `next.config.js` file

<details>
  <summary>`next.config.js`</summary>

```js
# Paste content of your `next.config.js` file here. Check there is no private info in there.
```

</details>

### Builds logs (or link to your logs)

<details>
  <summary>Build logs</summary>

```
# Paste logs here
```

</details>

### Function logs

<details>
  <summary>Function logs</summary>

```
# Paste logs here
```

</details>

### .next JSON files

<details>
  <summary>generated .next JSON files</summary>

```
# Paste file contents here. Please check there isn't any private info in them
# You can either build locally, or download the deploy from Netlify by clicking the arrow next to the deploy time.
```

</details>

**Comment by Kemystra** [comment]
Hello!

I encountered kinda the same problem with `netlify-cli` package. Apparently it tries to install `sharp@0.32.6`. If you try to install that manually, it will throw the same `Installation error: Request timed out`.

I'm still don't know specifically what package that depends on that library, because I don't see it in `netlify-cli`'s package.json. (On `netlify/next-runtime`, it is a direct dependency, but if you update the sharp dependency to use the latest version, you encounter another error. I dunno about this one, might be a setup error :|)

**Comment by Kemystra** [comment]
It seems like the `ipx` have just released its latest major version (v3.0.0) that uses a newer version of `sharp`. We just need to update the dependency to use the latest `ipx` version, although it might have breaking changes.
